config.js

```diff
@@ -16,7 +15,7 @@ const config = {
   },
   roles: {
     NOBOTRoleId: '783764176178774036',
-    adminRolesName: adminRoles,
+    adminRoles: ['core', 'maintainer', 'admin', 'moderator'],
```

Comment:
Agreed; although a deeper question better suited for a future issue: why do we actually use names and not id's for this

Reply:
> why do we actually use names and not id's for this

Not quite sure. I'm assuming copying all the role Ids would've been a hassle rather than just using role names?

Let's keep the names and `adminRolesNames` field name for now. What do you think?

Reply:
Agreed, think that works best. Will need to do a bit of magic on this PR to resolve some of the conflicts and then push an updated commit